software/console/console.py

```diff
@@ -0,0 +1,204 @@
+#
```

Comment:
the first line should be 

#!/bin/python3

the script should be named console and not console.py to keep everything else compatible


this allows the program to run by just typing 

./console 

if you are in the console directory or

path/to/console 

from everywhere else

Reply:
And used:
#!/bin/env python3